### ISSUE TITLE
Words "left" and "right" flipped in selectors before a comma 

### ISSUE THREAD

**Issue by NiviJah** [cssjanus/cssjanus#35]
Consider this example 

``` css
[class*="span"].pull-right,
.row-fluid [class*="span"].pull-right {
  float: right;
}
```

the code doesn't take the ","  as end of rule and therefor transform it as well.
I'm kindda new to Regex but I came up with this solution for now : 

``` javascript
lookAheadNotLetterPattern = '(?![a-zA-Z]|[\\,])'
```

**Comment by Krinkle** [comment]
@NiviJah A comma `,` cannot be the end of a rule in valid CSS. A rule is either ended explicitly with a semi-colon `;` or implicitly by the closing curly brace `}`.

What bug are you experiencing? The given example seems to flip without problems.

https://cssjanus.github.io/#input/%5Bclass*%3D%22span%22%5D.pull-right%2C%0A.row-fluid%20%5Bclass*%3D%22span%22%5D.pull-right%20%7B%0A%20%20float%3A%20right%3B%0A%7D
```css
/* CSSJanus output */

[class*="span"].pull-left,
.row-fluid [class*="span"].pull-right {
  float: left;
}
````

**Comment by YairRand** [comment]
@Krinkle In the example, the first use of the selector ".pull-right" is being flipped to ".pull-left", which is not the typical or expected behaviour. (Note that this is fixed in #36 .)

**Comment by Krinkle** [comment]
@YairRand Thanks, I did see that being flipped but assumed that was intentional.

But then I noticed that we don't have explicit code for that indeed. And more importantly, we currently only flip the words "left" and "right" in selectors _before_ a comma.

```css
.foo-left,
.bar-left,
.baz-left { }
```

This would currently flip `foo` and `bar` but not `baz`. That is indeed a bug, and I agree that the fix should be to not alter the class names.
